src/vasp/postprocessing/postprocessing_h5py/create_hi_pass_viz.py

```diff
@@ -564,7 +566,7 @@ def main():
 
     # In case of strain, modify the path to the solid mesh as the strain might be
     # computed for the solid domain only
-    if quantity == "strain":
+    if quantity == "strain" or "stress":
```

Comment:
There is a minor bug here since `stress` will always be `True`. Instead, it should be 
``` python
if quantity in {"strain", "stress"}:
```